src/features/missions/missionSlice.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable max-len */
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+// Get missions from API
+export const getMission = createAsyncThunk('mission/getMission', async () => {
+  const result = await axios('https://api.spacexdata.com/v3/missions');
```

Comment:
[OPTIONAL] While you did an amazing job working on the project requirements I noticed you  disabled eslint it is against microverse policy to disable linters because linters help us write clean code. So it would be nice if you enabled eslint.

Reply:
Thanks for this review, I would enable the linters and fix the code correctly.